packages/components/src/progress-bar/style.scss

```diff
@@ -0,0 +1,43 @@
+.components-progress-bar {
```

Comment:
In general, for new components we'd prefer Emotion over Sass if only so we don't expose any CSS classnames. (Emotion exodus on the horizon notwithstanding 😄)

Reply:
Good point. I've refactored it to Emotion in 166ae8d0fce590cbf2a2e55797e5f768a11d114e. 

I'm keeping the root class name though, because I thought it might be useful regardless - let me know if the recommended best practice is to remove it as well. Although I can definitely think of custom UIs that would want to alter it without utilizing Emotion.